components/Blog/useBlogApi.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```

Comment:
Please remove this file and refactor to use a local static JSON file of blog posts. This logic is not intuitive, and is not strictly necessary this early on in the blog feature implementation.